resources/public/api/conf/1.0/errors.raml

```diff
@@ -12,7 +12,7 @@ traits:
             type: types.errorResponse
```

Comment:
The error descriptions in this file are quite old. Can you update them to match the spec for retrieve coding out underpayments please?